src/CSVMsg.chpl

```diff
@@ -78,14 +78,18 @@ module CSVMsg {
 
     }
 
-    proc prepFiles(filename: string, overwrite: bool, A) throws { 
+    proc prepFiles(filename: string, overwrite: bool, A) throws {
+        // TODO can prob make prefix and extension const
         var prefix: string;
         var extension: string;
         (prefix,extension) = getFileMetadata(filename);
```

Comment:
Ya, this should be able to be rewritten as `const (prefix,extension) = getFileMetadata(filename)`, I don't think there is a reason for this to be 3 lines

Reply:
i'll go ahead and add that!